architecture/management_console.adoc

```diff
@@ -5,17 +5,17 @@
 :icons:
 :experimental:
 :toc: macro
-:toc-title: 
+:toc-title:
 
 toc::[]
 
 == Overview
-The OpenShift Management Console is a web-based front end for using OpenShift.  It is started as part of the link:../installation/rpm.html#starting-the-openshift-master[master] and by default listens one port higher than the API server.  All static assets required to run the console are served from the `openshift` binary.
+The OpenShift Management Console is a web-based front end for using OpenShift. It is started as part of the link:../installation/rpm.html#starting-the-openshift-master[master] and by default listens one port higher than the API server. All static assets required to run the console are served from the `openshift` binary.
```

Comment:
Re: `link:../installation/rpm.html#starting-the-openshift-master[master]`

This target no longer exists. Could you update this link to point to maybe `kubernetes_infrastructure.html#master`?